ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -447,7 +447,7 @@ export const ProfilePage = withUserProfile()(class extends React.Component<
                   valueKey: 'verifiedInstitutionalAffiliation.institutionDisplayName',
                   disabled: true
                 })}
-                {requireInstitutionalVerification && !profile.verifiedInstitutionalAffiliation &&
+                {!profile.verifiedInstitutionalAffiliation &&
```

Comment:
probably should have flag-protected the section above this.  So it's been always-true for a while already.  oops.